gadget/install/partition.go

```diff
@@ -177,42 +177,55 @@ func deviceName(name string, index int) string {
 
 // removeCreatedPartitions removes partitions added during a previous install.
 func removeCreatedPartitions(lv *gadget.LaidOutVolume, dl *gadget.OnDiskVolume) error {
-	indexes := make([]string, 0, len(dl.Structure))
+	sfDiskindexes := make([]string, 0, len(dl.Structure))
+	deletedIndexes := make(map[int]bool, 4)
```

Comment:
Nitpick, this could use a comment explaining the origins of number 4 (AFAIU it's the number of partitions we have with uc20); or if this number can be inferred from other pre-defined data then use a self-documenting expression.

Reply:
Sure good point

Reply:
done